### ISSUE TITLE
Floating window logs

### ISSUE THREAD

**Issue by plusCubed** [plusCubed/matlog#2]

**Comment by SnowVolf** [comment]
Can you explain this in more detailed form?

**Comment by plusCubed** [comment]
Yes - display the real-time log in a floating window on top of other apps, that way when you interact with them you immediately see the logcat.

(Thanks a ton for maintaining this BTW, I've been super busy. Let me know if/when you would like me to make a new release.)
